topics/admin/tutorials/empathy/slides.html

```diff
@@ -0,0 +1,134 @@
+---
+layout: base_slides
+logo: "GTN"
+
+title: "Empathy"
+contributors:
+  - erasche
+---
+
+## Part of the Job
+
+We are not just janitors / code monkeys
+- We're humans with feelings
+- Our users are humans with feelings as well
+- Both groups need to be understanding of each other
+
+We're (often) the point of contact for any problems
+- It's our responsibilty to fix Galaxy problems
+- Or any server problems for our usres
+- Of sometimes anything that is electrical is our problem
```

Comment:
?? Sometimes things that aren't meant to be our problem become ours ??

Reply:
Or is this about Electrical supply?

Reply:
```suggestion
- Or sometimes anything that is electrical is our problem
```